### ISSUE TITLE
Building problems on OSX

### ISSUE THREAD

**Issue by bvssvni** [tomaka/glutin#373]
`rustc 1.0.0-nightly (6b95d8bed 2015-04-09) (built 2015-04-09)`

```
/Users/sven/.cargo/registry/src/github.com-1ecc6299db9ec823/objc-0.1.2/src/runtime.rs:390:45: 390:49 error: cannot borrow `*self` as mutable because it is also borrowed as immutable
/Users/sven/.cargo/registry/src/github.com-1ecc6299db9ec823/objc-0.1.2/src/runtime.rs:390                 let self_ptr: *mut Object = self;
                                                                                                                                      ^~~~
/Users/sven/.cargo/registry/src/github.com-1ecc6299db9ec823/objc-0.1.2/src/runtime.rs:385:19: 385:23 note: previous borrow of `*self` occurs here; the immutable borrow prevents subsequent moves or mutable borrows of `*self` until the borrow ends
/Users/sven/.cargo/registry/src/github.com-1ecc6299db9ec823/objc-0.1.2/src/runtime.rs:385         let cls = self.class();
                                                                                                            ^~~~
/Users/sven/.cargo/registry/src/github.com-1ecc6299db9ec823/objc-0.1.2/src/runtime.rs:396:6: 396:6 note: previous borrow ends here
/Users/sven/.cargo/registry/src/github.com-1ecc6299db9ec823/objc-0.1.2/src/runtime.rs:384             where T: Encode {
...
/Users/sven/.cargo/registry/src/github.com-1ecc6299db9ec823/objc-0.1.2/src/runtime.rs:396     }
                                                                                              ^
error: aborting due to previous error
Build failed, waiting for other jobs to finish...
Could not compile `objc`.
```

**Comment by bvssvni** [comment]
I've opened up a PR to fix it https://github.com/SSheldon/rust-objc/pull/23

**Comment by SSheldon** [comment]
Just merged SSheldon/rust-objc#23 and published objc 0.1.3, should be fixed by cargo updating. Thanks @bvssvni!
